.github/workflows/ci.yml

```
@@ -24,15 +24,17 @@ jobs:
       with:
         github-access-token: ${{ secrets.NIX_GIT_TOKEN }}
 
+    # - flox develop does not support non-interactive shells at the moment
+    # - flox commands have command order isssues, to be fixed here in runix
+    #   and perculate to the install-flox-action
```

```suggestion
    #   and percolate to the install-flox-action
```

What are the command order issues?